packages/client/tests/functional/extensions/enabled/query.ts

```diff
@@ -0,0 +1,832 @@
+import { randomBytes } from 'crypto'
+import { expectTypeOf } from 'expect-type'
+
+import { wait } from '../../_utils/tests/wait'
+import { waitFor } from '../../_utils/tests/waitFor'
+import { NewPrismaClient } from '../../_utils/types'
+import testMatrix from './_matrix'
+// @ts-ignore
+import type { PrismaClient } from './node_modules/@prisma/client'
+
+let prisma: PrismaClient<{ log: [{ emit: 'event'; level: 'query' }] }>
+declare const newPrismaClient: NewPrismaClient<typeof PrismaClient>
+
+const randomId1 = randomBytes(12).toString('hex')
+const randomId2 = randomBytes(12).toString('hex')
+const randomId3 = randomBytes(12).toString('hex')
+
+jest.retryTimes(3)
```

Comment:
I see below you are using `waitFor`. Do we still need `retryTimes` in that case?